### DISCUSSION TITLE
Rendering support for Vision Pro?

### DISCUSSION THREAD

**Discussion by cpheinrich** [google/filament#6897] [Ideas]
It looks like Apple's Vision Pro will support custom renderers via Metal APIs if viewing 3D content in a fully immersive space, as discussed in this session: https://developer.apple.com/videos/play/wwdc2023/10089/

While there are still a number of questions about the APIs and hardware, I'm just curious if adding a stereoscopic Vision Pro renderer is something that would be on the Filament team's roadmap. If not, we may pursue developing it ourselves, or could collaborate on it, either way curious what the thoughts are.

**Comment by cpheinrich** [comment]
@romainguy any comment on this ?

**Comment by romainguy** [comment]
VR (as in stereoscopic rendering) is something we'd like to have. Targeting Vision Pro specifically is not on our roadmap but a generic solution would be great (cc @pixelflinger @bejado @poweifeng). There might be existing work happening in that space from other contributors so we should check before we tell you to do it/collaborate on it.
